sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusHostBuilderExtensions.cs

```diff
@@ -85,6 +85,16 @@ public static IWebJobsBuilder AddServiceBus(this IWebJobsBuilder builder, Action
 
                     section.Bind(options);
 
+                    if (options.MinMessageBatchSize > options.MaxMessageBatchSize)
+                    {
+                        throw new InvalidOperationException("The minimum message batch size must be less than the maximum message batch size");
+                    }
+
+                    if (options.MaxWaitTime > TimeSpan.FromSeconds(150))
+                    {
+                        throw new InvalidOperationException("The maximum wait time must be less than 2 minutes and 30 seconds.");
```

Comment:
We may want to offer context as to why.   The 10 minute time in Event Hubs was kind of arbitrary - but this is rooted in lock renewal/expiration, no?

Reply:
Yes it is, I added the documentation from the property into the exception as well